afm/server.py

```diff
@@ -96,8 +96,10 @@ def _get_locations(self, workers):
         return locations
 
     def get_flight_info(self, context, descriptor):
-        logger.info('get_flight_info: command={}'.format(descriptor.command))
         cmd = AFMCommand(descriptor.command)
+        logger.info('getting flight information',
+            extra={'command': descriptor.command,
+                   'AssetID': cmd.asset_name})
```

Comment:
Please use the same key as defined in the manager in logging.go.  Please make it a constant in logging.py.
If you can think of a way to define constants across languages - go, python, and java ... that would be even better.  :)